mne/tests/test_annotations.py

```diff
@@ -582,6 +582,17 @@ def test_events_from_annot_in_raw_objects():
     with pytest.raises(ValueError, match='not find any of the events'):
         events_from_annotations(raw, regexp='not_there')
 
+    # concat does not introduce BAD or EDGE
+    raw_concat = concatenate_raws([raw.copy(), raw.copy()])
+    _, event_id = events_from_annotations(raw_concat)
+    assert isinstance(event_id, dict)
+    assert len(event_id) > 0
+    for kind in ('BAD', 'EDGE'):
+        assert '%s boundary' % kind in raw_concat.annotations.description
+        for key in event_id.keys():
+            assert kind not in key
```

Comment:
@massich yes every time I find a failure on CircleCI I try to unit-test-ize it. This is where it's done here. (The underlying problem for those examples was that `BAD` and `EDGE` events were being added, and that `event_id` wasn't being used -- the former is tested here, the latter is actually just an example scripting error.)